nix/launcher-config.nix

```diff
@@ -96,25 +96,46 @@ let
     path.macos64 = "${dataDir}/Logs";
   in path.${os};
 
+  tlsConfig = {
+    ca = {
+      organization = "Daedalus";
+      commonName = "Daedalus Self-Signed Root CA";
+      expiryDays = 3650;
```

Comment:
@disassembler why is this set to 3650 days and others are 365 days?

Reply:
Convention is CA expiry is longer than cert expiry. In this case it really doesn't matter as we don't save the ca key.

Reply:
Got it!